CI/test_mdsuite.py

```diff
@@ -0,0 +1,5 @@
+from mdsuite import __version__
+
+
+def test_version():
```

Comment:
What is the protocol for automatics updates here? Or is this going to need to be added to a release pipeline of sorts.